aat/src/aat/resources/features/F-106 New CategoryID Column/S-106.3.td.json

```diff
@@ -0,0 +1,580 @@
+{
+  "_guid_": "S-106.3",
+  "_extends_": "Get_CaseType",
+  "title": "Import Definition file with correctly configured CategoryID in the CaseField tab",
+  "specs": [
+    "an active profile in CCD",
+    "contains filled in CategoryID for Document value",
+    "contains filled in CategoryID for Document Collection value",
+    "contains null CategoryID for Document value",
+    "contains null CategoryID for Document Collection value"
+  ],
+  "request": {
+    "pathVariables": {
+      "id": "FT_CaseFileView"
+    }
+  },
+  "expectedResponse": {
+    "body": {
+      "id": "FT_CaseFileView",
+      "description": "Case Type for testing CaseFileViews",
+      "version": {
+        "number": "[[ANY_INTEGER_NOT_NULLABLE]]",
+        "live_from": "[[ANYTHING_PRESENT]]",
+        "live_until": null
+      },
+      "name": "[[ANYTHING_PRESENT]]",
+      "jurisdiction": "[[ANYTHING_PRESENT]]",
+      "events" : "[[ANYTHING_PRESENT]]",
+      "states": "[[ANYTHING_PRESENT]]",
+      "acls": "[[ANYTHING_PRESENT]]",
+      "searchAliasFields": [],
+      "roleToAccessProfiles": "[[ANYTHING_PRESENT]]",
+      "searchParties": [],
+      "searchCriterias": [],
+      "case_fields" : [ {
```

Comment:
Can we add an `__ordering__` statement just to be on the safe side as `DEFAULT_COLLECTION_ASSERTION_MODE` isn't set on def store.

```
{
   "__ordering__": "UNORDERED",
   "__elementId__": "id"
 },
```

Reply:
If you see an y others in the JSON feel free to adjust those.  e.g. CategorisedDocumentComplexType.field_type.complex_fields.